test/test_helpers/backend.rb

```diff
@@ -85,7 +85,7 @@ def raise_on_method(klass, *methods)
     MockCore.mock_core!
 
     def stub_core_integration_errors(service_id: )
-      MockCore.stubs.get("/internal/services/#{service_id}/errors/", {'User-Agent' => 'Faraday v0.9.2'}) do
+      MockCore.stubs.get("/internal/services/#{service_id}/errors/", {'User-Agent' => 'Faraday v0.12.2'}) do
```

Comment:
Do we need this header?
I think it does not make any harm to remove it. @3scale/system

Reply:
This is for tests 🤔 We can try to remove it and if all tests pass, all good 😄

Reply:
I think Faraday sets it accordingly when omitted: https://github.com/lostisland/faraday/blob/v0.12.2/lib/faraday/connection.rb#L102

Now we could also use `Faraday.default_connection_options` if we're not really aiming to assert anything that could differ based on the User-Agent: https://stackoverflow.com/questions/45622842/globally-configure-faraday-user-agent

Reply:
Well I do not see the point of testing this header. 
If it was on purpose then I do not know the purpose of it.

Reply:
Tests fail if a fixed User-Agent is provided and it mismatches the actual Faraday version.